vagrant/k8s/Vagrantfile

```diff
@@ -98,6 +98,18 @@ SCRIPT
 cluster = read_cluster_config
 create_etc_hosts(cluster)
 
+def customize(v)
+  # make all nics 'virtio' to take benefit of builtin vlan tag
+  # support, which otherwise needs to be enabled in Intel drivers,
+  # which are used by default by virtualbox
+  # changes the settings for the first 5 NICs, regardless of presence
+  1.upto(5) do |n|
```

Comment:
https://github.com/contiv/netplugin/blob/master/Vagrantfile#L202 is exactly the same. We've been using this for a while in the main Vagrantfile. That Vagrantfile has been used hundreds of times since that PR was merged.